spec/helpers/testUtils.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
```

Comment:
I think we should add eslint to this repo and make sure to follow the same standards as the cumulus repo, like documenting functions.